### DISCUSSION TITLE
極端高溫產生的新職業：熱浪長🌡️

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#827] [General]
**新聞來源:**
https://technews.tw/2022/10/24/what-is-a-chief-heat-officer/

**內容大綱:**
各國發現，極端高溫比傳統災害還致命。美國過去10年因高溫死亡人數，比颶風死亡人數多14倍。
2021年4月，美國城市邁阿密聘任全球第一個熱浪長，後來美國多城市、希臘雅典、墨西哥蒙特雷等各地城市也開始設立，專責解決氣候變遷導致的高溫問題。
目前各地熱浪長有共識的目標是，增加城市「冷卻設施」，尤其是迷你公園和綠帶。
因此，各地熱浪長優先任務就是讓城市平均散布綠蔭避難所，保護不得已必須在沒有冷氣的戶外謀生或活動的人。

**心得感想:**
現在因為氣候變遷的速度太快，造成很多地區因為熱浪而造成傷亡
戶外活動也更需要注意高溫的曝曬，避免中暑或是熱衰竭的發生!
我覺得熱浪長的設立或許能提高民眾對於地球氣候的意識，並且改善人類生存的環境跟地球現況!
![image](https://user-images.githubusercontent.com/90883526/197971573-93ab61b7-f700-4886-9dde-e04c1d56fa0d.png)

**Comment by moonnight3503** [comment]
現在年均溫逐年增長，造成地球多處冰層融化，也有多數地區不適合人居住，但我也有聽過這可能是正常的現象，未來可能會面臨到冰河時期，然後溫度降低。

**Reply by bonny123865** [reply]
希望如此!可能這也是地球環境的必經過程?!

**Comment by ru0372572** [comment]
雖然有一派是說溫度升高是週期變化，但從二氧化碳濃度與溫度歷年的關係圖來看，人類活動確實加快了地球增溫的速度。

**Reply by bonny123865** [reply]
好像自從工業革命開始，人類發展的速度跟破壞環境的速度快成正比了
希望未來能漸漸彌補對環境造成的傷害

**Comment by CPCpyt** [comment]
我覺得地球暖化是需要全球共同努力的事，
但遺憾的是我們的經濟行為大多也都有賴於全球暖化。

**Comment by ruthchen1020** [comment]
我覺得人類跟環境是互相牽制又共生的，希望我們可以把永續發展當成接下來的發展目標之一。

**Comment by StevenWanglolz** [comment]
這個職也真的很重要，每次看到大熱天的還有老人家在大太陽底下賣東西真的覺得好辛苦，希望未來能幫助到這些人!

**Comment by Evan0979** [comment]
環境保護真的很重要,希望熱浪長的出現能為保護地球做出許多行動

**Comment by peggy0329** [comment]
這個新的公務員職位我認為很有幫助，甚至不只熱浪長，或許可以成立一個熱浪團隊，以支持政府對抗熱浪
除了熱浪，今年寒流也異常低溫，好多人被凍死
或許在極端氣候時期，也出現一個寒流長或許對許多人有幫助

**Comment by swerfg03joe** [comment]
這是一個天大的笑話，人類活動本身就是加劇暖化的元凶，工業革命後溫室氣體大量釋放，蓬勃的經濟發展帶動許多地區增長，接著就是人口的增加，人數上升則代表碳排放源的增加，不管用多好聽的理由說要發展綠能或其他永續能源，其過程必然伴隨許多的汙染，例如(太陽能)，如果真有心要控制極端氣候的發生，不如直接限制人口的增加，讓自然的修復速度超過人類的破壞速度才是正解
